### PR TITLE
[@mantine/core] (Multi)Select/Autocomplete : add positionDependencies

### PR DESCRIPTION
**Pull request by achmurali**

### CODE REVIEW

File: src/mantine-core/src/components/Select/Select.tsx [hunk 1]
```diff
@@ -87,6 +87,9 @@ export interface SelectSharedProps<Item, Value> {
 
   /** Whether to switch item order and keyboard navigation on dropdown position flip */
   switchDirectionOnFlip?: boolean;
+
+  /** useEffect dependencies to force update tooltip position */
```
Comment (rtivital) [inline src/mantine-core/src/components/Select/Select.tsx:91]: Invalid description

Comment (achmurali) [reply src/mantine-core/src/components/Select/Select.tsx:91]: Have followed this - https://github.com/mantinedev/mantine/blob/master/src/mantine-dates/src/components/DatePickerBase/DatePickerBase.tsx#L82, so should i update Datepicker's `positionDependencies` description?

Comment (rtivital) [reply src/mantine-core/src/components/Select/Select.tsx:91]: Yes

Comment (rtivital) [reply src/mantine-core/src/components/Select/Select.tsx:91]: It should be dropdown, not tooltip

**Comment by rtivital** [comment]
Thanks!

**Pull request merged**